session.go

```diff
@@ -116,11 +116,26 @@ func startSession(conn net.Conn, windowSize int, maxPadding int, pingInterval ti
 
 func (s *session) recvLoop() {
 	atomic.AddInt64(&recvLoops, 1)
+
+	stoppedOnEOF := false
+
 	defer func() {
 		atomic.AddInt64(&recvLoops, -1)
 		closeErr := s.Conn.Close()
 		if closeErr != nil {
-			log.Errorf("Error closing underlying connection: %v", closeErr)
+			logPrefix := "unexpected"
+			logFn := log.Errorf
+			if stoppedOnEOF {
```

Comment:
maybe `stoppedOnEOF && err == idletiming.ErrIdled`?

Reply:
Sorry, not sure what you're getting at?

Reply:
The way it's currently written we also don't log errors other than `idletiming.ErrIdled` as errors. IMU the goal was to filter out that particular error that started appearing so much on all proxies. There could theoretically be other errors on close that we would want to log as errors.

Reply:
Ah, I see what you're saying. Fixed.